test/unit/client-podsecuritypolicy.bats

```diff
@@ -129,3 +129,16 @@ load _helpers
       yq -c '.spec.hostPorts' | tee /dev/stderr)
   [ "${actual}" = '[{"min":8501,"max":8501},{"min":8502,"max":8502}]' ]
 }
+
+#--------------------------------------------------------------------
+# client.hostNetwork = true
+@test "client/PodSecurityPolicy: enabled with global.enablePodSecurityPolicies=true and hostNetwork=true" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -x templates/client-podsecuritypolicy.yaml  \
+      --set 'global.enablePodSecurityPolicies=true' \
+      --set 'client.hostNetwork=true' \
+      . | tee /dev/stderr |
+      yq '.spec.hostNetwork == true' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
```

Comment:
could you add another test checking that it's set to false by default. We didn't test it in the past because there was no templating logic, but now that there is I think it's worth adding a test.

Reply:
good catch, I'll add one for this, thanks!